src/librustc_resolve/resolve_imports.rs

```diff
@@ -562,8 +597,14 @@ impl<'a, 'b:'a, 'c: 'b> ImportResolver<'a, 'b, 'c> {
             // For better failure detection, pretend that the import will not define any names
             // while resolving its module path.
             directive.vis.set(ty::Visibility::Invisible);
-            let result = self.resolve_path(&directive.module_path[..], None, false,
-                                           directive.span, directive.crate_lint());
+            let result = self.resolve_path(
+                Some(ModuleOrUniformRoot::UniformRoot(keywords::Invalid.name())),
```

Comment:
What is the meaning of `UniformRoot` with `keywords::Invalid`?

Reply:
Ah, I see from the next commit.

Reply:
I'm open to better suggestions btw, using `keywords::Invalid` feels a bit hacky!